configs/model/synth.yaml

```
@@ -0,0 +1,58 @@
+
+MJCF_PATH: 'models/synth_model.xml'
+
+# Frames per clip for transform.
+N_FRAMES_PER_CLIP: 1
+
+# Tolerance for the optimizations of the full model, limb, and root. 
```

yamllint

[error] trailing spaces

(trailing-spaces)